1.7/administration/installing/high-availability-setup.md

```diff
@@ -0,0 +1,89 @@
+---
+post_title: Hybrid Cloud and Multi-Datacenter
+menu_order: 6
+---
+
+DC/OS supports multi-availability zone (AZ) configurations. This topic describes the setup recommendations and caveats.
+
+### Terminology
+
+#### Zone
+A zone is a failure domain that has isolated power, networking, and connectivity. Typically a single data center or independent fault domain on-premise or managed by a cloud provider. For example, AWS Availability Zones or GCP Zones. Servers within a zone are connected via high bandwidth (e.g. [1-10+ Gbps](https://blog.serverdensity.com/network-performance-aws-google-rackspace-softlayer/)), low-latency ([up to 1 ms](http://amistrongeryet.blogspot.com/2010/04/three-latency-anomalies.html)), and low-cost links.
```

Comment:
@vinodkone Are the two URLs here important to justify the bandwidth/latency numbers?  If they aren't, would the docs be clearer without them?

(Same with the URLs below.)